shortcircuit-api/src/main/java/com/powsybl/shortcircuit/json/FaultResultDeserializer.java

```diff
@@ -100,7 +109,9 @@ public FaultResult deserialize(JsonParser parser, DeserializationContext deseria
                     throw new AssertionError("Unexpected field: " + parser.getCurrentName());
             }
         }
-        FaultResult faultResult = new FaultResult(fault, shortCircuitPower, feederResults, limitViolations, current, voltage, shortCircuitBusResults, timeConstant);
+
```

Comment:
Add a check if status null and if there is no filled current, status is `FAILURE`, else `SUCCESS`